etc/install-zstd.sh

```diff
@@ -20,7 +20,7 @@ build_zstd() {
 
```

Comment:
Can we add errexit to this script? it will probably fail in subsequent steps but it could be left in an odd state